lms/static/js/spec/commerce/receipt_spec.js

```diff
@@ -0,0 +1,91 @@
+define([
+        'js/commerce/views/receipt_view'
+    ],
+    function (){
+        'use strict';
+        describe("edx.commerce.ReceiptView", function(ReceiptView) {
+            var view = null;
+            var data;
```

Comment:
There should only be 1 var declaration in a function, comma separated for multiple variables.